components/function-runtimes/nodejs12/kubeless/kubeless_rt/lib/tracer.js

```diff
@@ -3,79 +3,95 @@
 const axios = require('axios');
 
 const opentelemetry = require('@opentelemetry/api');
-const { registerInstrumentations } = require('@opentelemetry/instrumentation');
-const { NodeTracerProvider } = require('@opentelemetry/node');
-const { SimpleSpanProcessor } = require('@opentelemetry/tracing');
-const { OTLPTraceExporter } =  require('@opentelemetry/exporter-trace-otlp-grpc');
-const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
-const { B3Propagator,B3InjectEncoding } = require("@opentelemetry/propagator-b3");
-const  {NoopTracerProvider} = require('@opentelemetry/api')
-const {ExpressInstrumentation} = require("@opentelemetry/instrumentation-express");
+const { registerInstrumentations } = require( '@opentelemetry/instrumentation');
+const { NodeTracerProvider } = require( '@opentelemetry/sdk-trace-node');
+const { SimpleSpanProcessor, SamplingDecision } = require( '@opentelemetry/sdk-trace-base');
+const { OTLPTraceExporter } =  require('@opentelemetry/exporter-trace-otlp-http');
 const { Resource } = require( '@opentelemetry/resources');
+const { B3Propagator, B3InjectEncoding } = require("@opentelemetry/propagator-b3");
+const { ExpressInstrumentation, ExpressLayerType } = require( '@opentelemetry/instrumentation-express');
+const { HttpInstrumentation } = require('@opentelemetry/instrumentation-http');
 
-const TRACER_SAMPLE_HEADER= "x-b3-sampled"
+const ignoredTargets = [
+  "/healthz", "/favicon.ico", "/metrics"
```

Comment:
Why do we have to add the `/favicon.ico` endpoint? Previously we didn't have this one

Reply:
it should be there.. otherwise when function URL is used from a browser the requests to favicon is being traced